api/services/ocean/gke/schemas/ocean-compute.yaml

```diff
@@ -7,7 +7,7 @@ required:
   - availabilityZones
 properties:
   networkInterfaces:
-    type: array
+    type: object
```

Comment:
why? how did you check this?

Reply:
Line 16 uses "properties" which is used with the object data type. If it was an array then it would use items. Also our old documentation shows that it is an object data type: http://help.dev.spotinst.com/spotinst-api/ocean/ocean-cloud-api/ocean-for-gke/create/